docs/requirements.txt

```diff
@@ -2,3 +2,8 @@ pyenchant
 Sphinx>=4.5.0
 sphinxcontrib-spelling
 blacken-docs
+
+# Due to the viewcode extension, the docs are dependent on django itself.
+# Below we install django from the local directory. This assumes installation
+# from the project root i.e. `python -m pip install -r docs/requirements.txt`
```

Comment:
(missing period to end the comment)... actually I first thought the period on the next line was for this sentence.